modules/management/locals.tf

```diff
@@ -233,7 +233,7 @@ locals {
     "${local.root_id}-landing-zones" = {
       parameters = {
         Deploy-AzSqlDb-Auditing = {
-          logAnalyticsWorkspaceId = local.log_analytics_workspace_resource_id
+          logAnalyticsWorkspaceId = lower(local.log_analytics_workspace_resource_id)
```

Comment:
@matt-FFFFFF It turns out this local just overrides the settings from the policy assignment archetype anyway, so may as well just have been an empty string in the other two files updated here...

Reply:
Could we just have this in as the workaround? Leaving PascalCase for everything else?

Reply:
As with the other comment, this is literally just used by that one policy. All others construct the id based on a workspace name passed in.